extensions/import/src/test/import.test.ts

```diff
@@ -178,11 +222,99 @@ describe('import extension tests', function (): void {
 			mockApiWrapper.verify(x => x.showErrorMessage(TypeMoq.It.isAny()), TypeMoq.Times.once());
 
 		});
+
+		it('FlatFileWizard- getConnection returns active connection', async () => {
```

Comment:
I would strongly prefer that instead of having one massive test file you split the tests into a separate test file corresponding to each source file. This helps makes it easier to find tests corresponding to a specific component and keeps things better organized. 

See https://github.com/microsoft/azuredatastudio/blob/main/extensions/azurecore/src/test/ as an example